third_party/gulp-eslint/README.md

```diff
@@ -0,0 +1,5 @@
+This fork modernizes [gulp-eslint](https://github.com/adametry/gulp-eslint) to use the latest ESLint version.
```

Comment:
@mmmicedcoffee Hopefully this README helps explain the situation more

Reply:
Thanks!